src/catch2/catch_session.hpp

```diff
@@ -16,6 +16,13 @@
 
 namespace Catch {
 
+    constexpr int TestFailureExitCode = 42;
```

Comment:
I believe these constants will be defined separately in each translation unit leading to marginally higher memory usage but I don't believe it will cause any linker errors so it should function exactly as expected. I'd prefer to use C++17's `inline` here but that's out of reach so long as our minimum C++ standard is C++14.

Reply:
This is _fiiiiiine_.

Worth adding a note that these can be upgraded to `inline` constexpr vars with C++17.